doc/input-output-reference/src/overview/group-unitary-equipment.tex

```diff
@@ -2032,11 +2032,11 @@ \subsubsection{Outputs}\label{outputs-3-021}
 
 \paragraph{Unitary System Electricity Rate {[}W{]}}\label{unitary-system-electric-power-w-1}
 
-This output field is the electricity consumption rate of the multispeed heat pump in Watts. The consumption includes electricity used by the DX coils (including crankcase heater if the fuel type is electricity), fans (indoor supply air fan and the condenser fans associated with the DX coil{[}s{]}), auxiliary power during on and off period, and the supplemental heating coil (if electric). This value is calculated for each HVAC system timestep being simulated, and the results are averaged for the timestep being reported. Any non-electric energy use is not reported by the heat pump object but is reported in the associated coil objects as appropriate.
+This output field is the electricity consumption rate of the multispeed heat pump in Watts. The consumption includes electricity used by the DX coils (including crankcase heater if the fuel type is electricity), fans (indoor supply air fan and the condenser fans associated with the DX coil{[}s{]}), defrost mode operation (resistive or reverse-cycle), auxiliary power during on and off period, and the supplemental heating coil (if electric). This value is calculated for each HVAC system timestep being simulated, and the results are averaged for the timestep being reported. Any non-electric energy use is not reported by the heat pump object but is reported in the associated coil objects as appropriate.
 
```

Comment:
I/O Ref Documentation update that the parent `electricity rate` included `defrost electric power`.